src/DDTrace/Configuration.php

```diff
@@ -67,7 +67,7 @@ public function isAnalyticsEnabled()
     public function isPrioritySamplingEnabled()
     {
         return $this->isDistributedTracingEnabled()
```

Comment:
The distributed tracing check exists in `ddtrace_config_priority_sampling_enabled()` so we should be able to remove the `$this->isDistributedTracingEnabled()` check.